src/widget/wwidgetgroup.cpp

```diff
@@ -152,6 +149,15 @@ void WWidgetGroup::setup(const QDomNode& node, const SkinContext& context) {
 
     if (pLayout) {
         setLayout(pLayout);
+
+        if (layoutIsStacked) {
+            // Adding a zero-size dummy widget as index 0 here before
+            // any child is added in the xml template works around
+            // https://bugs.launchpad.net/mixxx/+bug/1627859
```

Comment:
Is this still needed? If so, can we use the  GitHub issue URL?

Reply:
@Holzhaus according to @ronso0 (see zulipchat) "The workaround is still required for Qt 5.12.8"

Where do I find the github issue URL ?

Reply:
After migrating to Github Issues the new location was posted to each migrated launchpad bug.
So, last comment in https://bugs.launchpad.net/mixxx/+bug/1627859:
https://github.com/mixxxdj/mixxx/issues/8655

Reply:
we could as well improve the comment:
```
// Adding a zero-size dummy widget as index 0 here before
// Without this zero-sized dummy widget being added before
// any child widgets, the stacked layout would force-show
// the top item, i.e. ignore the state of its 'visible' connection.
// See https://github.com/mixxxdj/mixxx/issues/8655
```